onboarding_documentation/technical_documentation/analyst_tutorial/analyst_tutorial.md

```diff
@@ -0,0 +1,122 @@
+
+<details>
+<summary>Click to reveal the config file</summary>
+
+```TOML
+[workflow]
+input_datasets = ['bioheart-test']
+sequencing_type = 'genome'
+output_version = '1.0' # Do we want them to specify output_version?
+only_sgs = [<list of sequencingGroup IDs>] # to be used to demonstrate how to run on a subset of samples
+```
+
+</details>
+
+Once the config file has been written, save it with an appropriate name (e.g. `bioheart-test.toml`) in a directory that makes sense on your local machine. In the following step we will be using `analysis-runner` to submit the job and point it to the config file we have just written.
+
+## 3. Submitting an analysis job
+Now that we have a config file, we can submit an analysis job. To do this we will use the `analysis-runner` tool. The `analysis-runner` tool is a command line tool that is used to submit analysis jobs to the cloud.
+
+Because we are running the `large cohort` pipeline we will need to be inside the `Production-pipelines` repository. So in a directory you want to keep `Production-pipelines` in, run the following command:
+
+```bash
+git clone --recurse-submodules git@github.com:populationgenomics/production-pipelines.git
+cd production-pipelines
+```
+
+Ensure you are on the `main` branch.
+
+For guidance on how to use the `analysis-runner` tool, please see the `analysis-runner` documentation [here](https://github.com/populationgenomics/analysis-runner). It will need to be `pip` installed before use.
+
+It is generally good practice to create a bash script that contains the `analysis-runner` command to submit the job. This is so that the command can be easily re-run if necessary.
+
+#### Task: Submit an analysis job
+- Using the instructions in the `analysis-runner` repo linked above as well as the config file you have just written, write the command to submit analysis job to the cloud using the `analysis-runner` tool.
+
+<details>
+<summary>Click to reveal the command</summary>
+
+```bash
+analysis-runner \
+--dataset bioheart-test \
+--description "Running large cohort up to Combiner stage on bioheart" \
+--output-dir "bioheart-test-YOUR-NAME" \
+--access-level test \
+--config configs/genome.toml \
+--config configs/defaults/large_cohort.toml \
+--config path/to/you/config/config_filename.toml \
+--image australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_workflows:latest \
+main.py large_cohort
+```
+</details>
+
+
+
+## 4. Analysing the output
+
+Now that we have some output to work with, lets begin analysing.
+
+Analysis is best done within a Jupyter Notebook. However, in order to interact with our data in the cloud using Hail and jupyter notebooks we need to do some set up first. 
+
+Instructions on setting up a Jupyter Notebook in the cloud can be found [here](https://github.com/populationgenomics/team-docs/blob/main/notebooks.md)
+
+Please continue this tutorial once you have a Jupyter Notebook running in the cloud, good luck!
```

Comment:
> good luck!

Again, I think a task-based tutorial with very little context and supporting help doesn't seem like the best way to help a new hire